labelbox/client.py

```diff
@@ -1408,3 +1415,30 @@ def get_model_slice(self, slice_id) -> ModelSlice:
         """
         res = self.execute(query_str, {"id": slice_id})
         return Entity.ModelSlice(self, res["getSavedQuery"])
+
+    def remove_feature_schema_from_ontology(self, ontology_id: str,
```

Comment:
please specify this function's return type

Reply:
[Updated](https://github.com/Labelbox/labelbox-python/pull/958/commits/42f7bed48bf8281f85c19093a2dd12b792526628)